### ISSUE TITLE
debugging a camera

### ISSUE THREAD

**Issue by dayne** [t3alliance/protero-cimex#1]
No pictures showing up?
Picture files you are taking are empty?

Camera setup sanity
- [ ] blue tape to protect from shorts?
- [ ] ribbon cable seated firmly with correct orientation (both sides)?
- [ ] camera enabled in `rpi-setup`

Sanity check list:
- [ ] [`raspistill`](https://www.raspberrypi.org/documentation/usage/camera/raspicam/raspistill.md) sanity check: `raspistill -v -o test.jpg` and view `test.jpg`

**Comment by clowrey** [comment]
was so satisfying to check those boxes lol

**Comment by clowrey** [comment]
@dayne are you planning to use this issue tracker as a tutorial of sorts? or debugging resource that users would search?

**Comment by dayne** [comment]
I want a place to capture debugging notes as we debug things -- more than once I've seen us providing the same debug information.

This issue will be closed when the information is captured and made available as a website for us to link folks to when debugging a camera.

**Comment by clowrey** [comment]
Yeah I agree with those points - what about using the github wiki?

Or adding the debugging information to the website camera lesson?

I'm sure there are benefits to each approach. I like using github but having it right in the lesson is important too. Maybe linking to the relevant wiki page from the website lesson or embedding it somehow?

**Comment by clowrey** [comment]
Using github issues as a way to help users would be powerful too - I wonder how easy that would be to get everyone to use. 

In my opinion its much easier to search and use as an index of information than searching back in Discord chats.
